airbyte-config/config-models/src/main/java/io/airbyte/config/CatalogDefinitionsConfig.java

```diff
@@ -15,7 +15,7 @@ public class CatalogDefinitionsConfig {
       SEED_SUBDIRECTORY + LOCAL_CONNECTOR_CATALOG_FILE_NAME;
 
   private static final String REMOTE_OSS_CATALOG_URL =
-      "https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/oss_catalog.json";
+      "https://connectors.airbyte.com/api/v0/catalog/oss_catalog.json";
```

Comment:
Im thinking we change this to the url that maps directly to the path.

While I like the API path, I think we can start using that at the time were uploading versions of these catalogs.

```suggestion
      "https://connectors.airbyte.com/oss_catalog.json";
```

Reply:
Either works! It's more for whatever backwards compatability you guys want to create for yourselves.